low-level/files/opt/seagate/sspl/conf/sspl_common_config.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+cluster:
+   minion_id: srvnode1
+   product:
+   setup: cortx
+   cluster_id: '001'
+   srvnode1:
+      operating_system: centos7
+      site_id: '001'
+      rack_id: '001'
+      node_id: '001'
+      type:
+
+      storage:
+         enclosure_id: enclosure_1
+
+      bmc:
+         ip:
+         secret: '****'
+         user:
+
+storage:
+   enclosure_1:
+      type:
+      controller:
+         primary_controller_ip: 10.0.0.2
+         primary_controller_port: '80'
+         secondary_controller_ip: 10.0.0.3
+         secondary_controller_port: '80'
+         user: manage
+         password: '****'
```

Comment:
***??
Keep some encrypted string as kept in sspl.conf

Reply:
Done